### PR TITLE
feat(nx-plugin): add create-package plugin

### PR DESCRIPTION
**Pull request by xiongemi**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/nx-plugin/src/generators/create-package/create-package.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+  return readJson(host, join(projectRoot, 'package.json'))?.name;
+}
+
+async function createCliPackage(
+  host: Tree,
+  options: NormalizedSchema,
+  pluginPackageName: string
+) {
+  addDependenciesToPackageJson(
```
Comment (AgentEnder) [inline packages/nx-plugin/src/generators/create-package/create-package.ts:55]: This task needs to be returned

File: packages/nx-plugin/src/generators/create-package/schema.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      "enum": ["jest", "none"],
+      "description": "Test runner to use for end to end (E2E) tests.",
+      "default": "jest"
+    },
+    "standaloneConfig": {
+      "description": "Split the project configuration into `<projectRoot>/project.json` rather than including it inside `workspace.json`.",
+      "type": "boolean",
+      "default": true,
+      "x-deprecated": "Nx only supports standaloneConfig"
+    },
```
Comment (AgentEnder) [inline packages/nx-plugin/src/generators/create-package/schema.json:72]: Lets not add this for new generators

**Review by AgentEnder — CHANGES_REQUESTED** [review]
This looks good, there are a few changes

File: packages/nx-plugin/src/generators/create-package/create-package.ts (hunk 1 above)
Comment (xiongemi) [reply packages/nx-plugin/src/generators/create-package/create-package.ts:55]: changed

**Pull request closed without merging**